core/src/components/textarea/textarea.ionic.outline.scss

```diff
@@ -0,0 +1,72 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./textarea.vars";
+
+// Textarea Fill: Outline
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) {
+  --border-width: #{globals.$ionic-border-size-025};
+  --border-color: #{globals.$ionic-color-neutral-500};
+}
+
+// Textarea Fill: Outline, Textarea Wrapper
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) .textarea-wrapper-inner {
+  /**
+   * The border should be relative to the inner wrapper
+   * so that it does not include the label.
+   */
+  position: relative;
+}
+
+// Textarea Fill: Outline, Outline Container
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) .textarea-outline {
+  @include position(0, 0, 0, 0);
+  @include border-radius(var(--border-radius));
+
+  position: absolute;
+
+  width: 100%;
+  height: 100%;
+
+  pointer-events: none;
+
+  border: var(--border-width) var(--border-style) var(--border-color);
+}
+
+// Textarea Fill: Outline, Bottom Content
+// ----------------------------------------------------------------
+
+/**
+ * The bottom content should never have
+ * a border with the outline style.
+ */
+:host(.textarea-fill-outline) .textarea-bottom {
+  border-top: none;
+}
+
+// Textarea Fill: Outline, Native Textarea
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) textarea {
+  margin-top: 4px;
+}
+
+// Textarea Fill: Outline, Start / End Slots
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) .start-slot-wrapper,
+:host(.textarea-fill-outline) .end-slot-wrapper {
+  margin-top: 3px;
+}
+
+:host(.textarea-fill-outline) .start-slot-wrapper {
+  margin-inline-start: 1px;
+}
+
+:host(.textarea-fill-outline) .end-slot-wrapper {
+  margin-inline-end: 1px;
```

Comment:
Same as above, but `ionic-border-size-025`

Reply:
I removed this per my comment here: https://github.com/ionic-team/ionic-framework/pull/29776#discussion_r1724010343